.github/workflows/build.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     - name: Install Rust
       uses: dtolnay/rust-toolchain@1482605bfc5719782e1267fd0c0cc350fe7646b8
       with:
-        toolchain:  1.71
+        toolchain:  1.76
```

Comment:
Is this really the only Github action this appears in? What about the Release action?

Reply:
No, no need, thanks for checking!